.github/workflows/release.yaml

```diff
@@ -111,8 +111,14 @@ jobs:
           directory: build
           githubToken: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Dotenv Action
```

Comment:
How would you otherwise make an optional, but still somewhat automated job to publish source maps to Sentry only when Sentry is actually enabled and active?

Otherwise, as per previous builds, Sentry will keep failing.

Reply:
I would rather you just mark the sentry bit as non-required for the workflow to be successful, so that even if it fails the workflow can still succeed.

Reply:
Instead of adding dotenv to the workflow, please just add [continue-on-error](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) to the sentry job.